pynumdiff/optimize/_optimize.py

```diff
@@ -65,8 +65,10 @@
               {'sigma': (1e-2, 1e3),
                 'lmbd': (1e-3, 0.5)}),
     tvrdiff: ({'gamma': [1e-2, 1e-1, 1, 10, 100, 1000],
-               'order': {1, 2, 3}}, # warning: order 1 hacks the loss function when tvgamma is used, tends to win but is usually suboptimal choice in terms of true RMSE
-              {'gamma': (1e-4, 1e7)}),
+               'order': {1, 2, 3}, # warning: order 1 hacks the loss function when tvgamma is used, tends to win but is usually suboptimal choice in terms of true RMSE
+              'huberM': [0., 1, 2, 3, 6]},
```

Comment:
Because the Huber parameter input is on the scale of scatter of input data rather than residuals, $\sigma$ is likely to be larger than it would be if we were able to subtract $y-x$, so it's necessary to more finely comb lower values here than for other `HuberM`s